### ISSUE TITLE
Startup fails when Bitbucket credentials are missing

### ISSUE THREAD

**Issue by mms-gianni** [kubero-dev/kubero#27]
Will fix this today

```
09:27:42 - Starting compilation in watch mode...
[0]
[1] socket.io started
[1] load kubectl config from file
[1] Gitlab API: undefined
[1] Gitlab token: glpat-.......
[1]  BITBUCKET CLOUD API LATEST UPDATES:  https://developer.atlassian.com/cloud/bitbucket
[1] /Users/carafagi/workdir/kubero-dev/kubero/node_modules/bitbucket/lib/index.umd.js:15
var r}function W(e){var r,t;return!1!==U(e)&&("function"==typeof(r=e.constructor)&&(!1!==U(t=r.prototype)&&!1!==t.hasOwnProperty("isPrototypeOf")))}function F(){var e=arguments.length>0&&void 0!==argum
```

**Comment by mms-gianni** [comment]
fixed here https://github.com/kubero-dev/kubero/commit/4fd1159e511498d101ac8a7987fca4a176198e80

**Comment by mms-gianni** [comment]
fixed in Version v1.3.1 

https://github.com/kubero-dev/kubero/releases/tag/v1.3.1
